chcemvediet/apps/anonymization/finalization.py

```diff
@@ -0,0 +1,85 @@
+import os
+import shutil
+import traceback
+
+import subprocess32
+from django.core.files.base import ContentFile
+from django.conf import settings
+
+from poleno.cron import cron_logger
+from poleno.utils.misc import guess_extension
+
+from .utils import temporary_directory
+from .models import AttachmentAnonymization, AttachmentFinalization
+from . import content_types
+
+
+LIBREOFFICE_TIMEOUT = 300
+
+def finalize_using_mock(attachment_finalization):
+    finalized = os.path.join(settings.PROJECT_PATH,
+                              u'chcemvediet/apps/anonymization/mocks/normalized.pdf')
```

Comment:
Pre mock finalizaciu nepouzivaj `normalized.pdf`, kedze to nie je OCRknute ale obrazok. Pdf vygenerovane z OCRknuteho mock suboru.

Reply:
:ok: